### DISCUSSION TITLE
how can i call a function?

### DISCUSSION THREAD

**Discussion by EIT0** [HaxeFlixel/flixel#2493] [Q&A]
```hx
{
    Shit.browserLoad('https://www.youtube.com/channel/UC4X_UAuj9BOpHgBHo8TvWoQ/featured');
    //want to call a function here
}
```

**Comment by Geokureli** [comment]
do you get errors when you try and call it? are you importing the necessary files for it? you're calling a function in the snippet, does that work?

**Reply by EIT0** [reply]
I already made it, but thanks a lot for you're help

**Reply by RichardBray** [reply]
How did you fix the issue?

**Reply by EIT0** [reply]
on Shit.hx after searching on the browser currletter goes back to 0
